walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/requirements/constraints/ClaimRegexConstraint.kt

```diff
@@ -0,0 +1,53 @@
+package com.microsoft.walletlibrary.requests.requirements.constraints
+
+import com.jayway.jsonpath.Configuration
+import com.jayway.jsonpath.JsonPath
+import com.jayway.jsonpath.Option
+import com.microsoft.did.sdk.credential.models.VerifiableCredentialContent
+import com.microsoft.did.sdk.util.log.SdkLog
+import com.microsoft.walletlibrary.util.VerifiedIdIssuerIsNotRequestedException
+import com.microsoft.walletlibrary.verifiedid.VerifiableCredential
+import com.microsoft.walletlibrary.verifiedid.VerifiedId
+import kotlinx.serialization.json.Json
+import java.util.regex.Pattern
+
+class ClaimRegexConstraint(internal val path: List<String>, internal val pattern: String) :
```

Comment:
nit: naming maybe something like VerifiableCredentialPathConstraint or something as this constraint only relates to a VC? what do you think @logangirvin

Reply:
I am struggling with naming for this class specifically since it is the same for claims and accepted issuer (which is not a claim). I like the name with Path and decouples claims relation.

Reply:
This specific class is only for Regex in the constraints. So, I add it to the name as well.